db_scripts/restore.sh

```diff
@@ -1,16 +1,24 @@
-
 #!/bin/bash
 
-# For AMI: sudo yum -y install https://download.postgresql.org/pub/repos/yum/9.6/redhat/rhel-6-x86_64/pgdg-ami201503-96-9.6-2.noarch.rpm
-#          sudo yum -y install postgresql96
+set -eu -o pipefail
+
+ENV_NAME=dev
+
+# aws cloudformation create-stack --region us-east-1 --stack-name $ENV_NAME-bastionhosts --template-url https://s3.amazonaws.com/crowd-deployment/infrastructure/bastion-hosts.yaml --parameters ParameterKey=EnvironmentName,ParameterValue=$ENV_NAME ParameterKey=KeyPairName,ParameterValue=rstorey@loc.gov --disable-rollback
+# aws cloudformation delete-stack --region us-east-1 --stack-name $ENV_NAME-bastionhosts
+if [ $ENV_NAME = "prod" ]; then
```

Comment:
This seems like the same question – with the earlier ENV_NAME assignment it seems like this will never be true.

Reply:
That's a good idea. I don't know why that didn't occur to me. I'll implement that the next time I use these scripts / next week.